crtool/jinja2/crtool/crt-start.html

```diff
@@ -235,12 +239,13 @@ <h2>Start a review</h2>
         <div class="o-tabs_tab o-tabs_tab__continue-review">
             <h2>Continue a review</h2>
             <p>Enter your access code to return to a review in progress.</p>
-            <form id="form__continue-review" class="o-form" method="GET" action="../tool">
+            <form id="form__continue-review" class="o-form" method="POST" action="../continue-review/">
+                {{ csrf_input }}
```

Comment:
This is now a regular form, submitting without JS, and redirects to `/tool/#id=abcd...` and out init() in the session already sets up everything in the browser that's missing.